maze4d/Config.h

```diff
@@ -19,9 +19,9 @@ class Config
 			WriteFile();
 			Log("Config not found, use default");
 		}
-		for each (std::pair<std::string, paramValues> param in PARAMS)
+		for (auto const& [paramName, paramValue] : PARAMS)
```

Comment:
I suggest using the old syntax here to be able to compile in VS2015.

Reply:
I changed loops to C++98 syntax. I hope this is enough for VS2015 (I cannot check it myself).

Reply:
Ok, thanks, it works.